pymongo/network_layer.py

```diff
@@ -18,32 +18,105 @@
 import asyncio
 import socket
 import struct
+import sys
+from asyncio import AbstractEventLoop, Future
+from ssl import SSLError, SSLSocket
```

Comment:
This has to be wrapped with try/except to account for when the ssl module is not available.